engines/mads/mps_installer.cpp

```diff
@@ -0,0 +1,160 @@
+/* ScummVM Tools
+ *
+ * ScummVM Tools is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "common/endian.h"
+#include "common/debug.h"
+#include "common/str.h"
+#include "common/file.h"
+#include "common/util.h"
+#include "common/memstream.h"
+#include "common/dcl.h"
+#include "mps_installer.h"
+
+namespace MADS {
+
+MpsInstaller* MpsInstaller::open(const Common::Path& baseName) {
+	Common::File indexFile;
+	Common::HashMap<Common::String, FileDescriptor, Common::IgnoreCase_Hash, Common::IgnoreCase_EqualTo> files;
```

Comment:
Use a typedef, and prefix the variable with _, as per our coding guidelines

```suggestion
	typedef Common::HashMap<Common::String, FileDescriptor, Common::IgnoreCase_Hash, Common::IgnoreCase_EqualTo> FileMap;
	
	FileMap _files;
```

Check https://github.com/scummvm/scummvm/blob/253a0519a7d403980d265739741155f3f27ba00d/common/installshieldv3_archive.h#L59